raiden/ui/web/src/app/services/network-type.enum.ts

```diff
@@ -0,0 +1,4 @@
+export const enum NetworkType {
+    MAIN = 1,
```

Comment:
Don't get tied to the py implementation here.. as you should receive `string` in `config.json`, make the [enum string-based](https://www.typescriptlang.org/docs/handbook/enums.html) according to the lowercase names